### PR TITLE
Added ms suffix to durations

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
Added ms suffix to durations to make sure it is clear the units of time

## Issues
N/A

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                   |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> | `73.754 kB`<br />`22.494 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                             |  `20.965 kB`<br />`6.605 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                               |  `21.918 kB`<br />`6.919 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                             |  `22.005 kB`<br />`6.582 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox</abbr>                                    | `60.564 kB`<br />`20.751 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                       |   `15.455 kB`<br />`5.57 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                             | `29.382 kB`<br />`10.047 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                   |   `13.61 kB`<br />`5.462 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                          |  `25.312 kB`<br />`8.202 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                    |   `20.672 kB`<br />`7.07 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8437eb3b6023bdac5b234538e1116b1c7d24f53f'>8437eb3b6023bdac5b234538e1116b1c7d24f53f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/Avatar-mount_base.html">930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/Avatar-mount.html">907</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/Button-mount_base.html">551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/Button-mount.html">559</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProvider-mount_base.html">1849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProvider-mount.html">1831</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProviderWithTheme-mount_base.html">275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProviderWithTheme-mount.html">286</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">217</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">239</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">314</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">315</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/MakeStyles-mount_base.html">1589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22831/perf-test-react-components/MakeStyles-mount.html">1598</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-theme/src/global/durations.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import type { DurationTokens } from '../types';
 
 export const durations: DurationTokens = {
-  durationUltraFast: '50',
```
[the remark this answers is not in the document]
Comment (miroslavstastny) [reply packages/react-components/react-theme/src/global/durations.ts:4]: https://github.com/microsoft/fluentui/blob/6c764b820241483c1a9a6e225f758a4c8d4b060e/packages/react-components/react-tabs/src/components/Tab/useTabAnimatedIndicator.ts#L24

this seems to be the only one in master, it is missing units

**Review by miroslavstastny — APPROVED** [review]

File: packages/react-components/react-theme/src/global/durations.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (GeoffCoxMSFT) [reply packages/react-components/react-theme/src/global/durations.ts:4]: By adding units to the token value then we don't need to update.  The default duration unit is milliseconds if you don't have a duration.

**Pull request merged**